redash/utils/parameterized_query.py

```diff
@@ -33,17 +33,59 @@ def _parameter_names(parameter_values):
     return names
 
 
+def _is_date(string):
+    try:
+        parse(string)
```

Comment:
We should parse based on a specific format.

Reply:
What's the risk here? A format is a data intake/display concern, not a security concern.